### PR TITLE
Fire installer failure on all failed results

### PR DESCRIPTION
**Pull request by M-Davies**
Signed-off-by: Morgan Davies <morgan.davies@ibm.com>

### CODE REVIEW

**Comment by M-Davies** [comment]
Follow up to #1717 which didn't work properly

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -192,8 +192,9 @@ class Build {
                             parameters: params
                     
                     // Output notification of downstream failure (the build will fail automatically)
-		    if (signJob.getResult() == 'FAILURE') {
-                        context.println "ERROR: downstream sign_build FAILED. See ${signJob.getAbsoluteUrl()}"
+					def jobResult = signJob.getResult()
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:195]: whitespace seems out for all of the changes.

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 2]
```diff
@@ -192,8 +192,9 @@ class Build {
...
+		    		if (jobResult != 'SUCCESS') {
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:196]: What other non-success error codes can you get?

Comment (M-Davies) [reply pipelines/build/common/openjdk_build_pipeline.groovy:196]: UNSTABLE and ABORTED

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (M-Davies) [reply pipelines/build/common/openjdk_build_pipeline.groovy:195]: Note to self, never use the github editor ever again. It was fine when I typed it 😭

**Pull request merged**